src/main/java/techcourse/myblog/web/ArticleController.java

```diff
@@ -2,10 +2,82 @@
 
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import techcourse.myblog.domain.Article;
 import techcourse.myblog.domain.ArticleRepository;
 
+import java.util.List;
+
 @Controller
 public class ArticleController {
     @Autowired
     private ArticleRepository articleRepository;
-}
+
+    @GetMapping("/")
+    private String getIndex(Model model) {
+        List<Article> articles = articleRepository.findAll();
+        model.addAttribute("articles", articles);
+        return "index";
+    }
+
+    @GetMapping("/articles/{id}")
+    private String getArticleById(@PathVariable int id, Model model) {
+        Article article = articleRepository.find(id);
+        model.addAttribute("id", id);
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        model.addAttribute("article", article);
```